src/appleseed/renderer/kernel/rendering/final/adaptivetilerenderer.cpp

```diff
@@ -454,10 +469,9 @@ namespace
             // Warn the user if adaptive sampling wasn't efficient on this tile.
             if (static_cast<float>(tile_converged_pixel) / static_cast<float>(pixel_count) < BlockConvergenceWarningThreshold / 100.0f)
             {
-                RENDERER_LOG_WARNING(
-                    "%s of pixels have converged, be sure to increase the maximum number of samples "
-                    "or decrease the noise threshold for better performance",
-                    pretty_percent(tile_converged_pixel, pixel_count, 1).c_str());
+                RENDERER_LOG_WARNING("%s of tile's pixels have converged, average sample/pixel: %s.",
+                    pretty_percent(tile_converged_pixel, pixel_count, 1).c_str(),
+                    pretty_uint(static_cast<size_t>(m_spp.get_mean())).c_str());
```

Comment:
Because it's sample/pixel, i used `pretty_scalar` with 1 digit but i think it makes more sense to have integer numbers. Per-pixel, we always sample a with an integer amount.